docs/zero-knowledge-glossary/index.mdx

```diff
@@ -87,7 +87,7 @@ In optimistic rollups, fraud proofs (fault proofs) are the evidence a verifier p
 
 gnark is a software library that offers users the ability to design and run computational circuits in zk-SNARKs through a high-level API.
 
-The library is open source and developed under the Apache 2.0 license.
+The library is open-source and developed under the Apache 2.0 license.
```

Comment:
```suggestion
The library is open source and developed under the Apache 2.0 license.
```

Hyphenation unnecessary unless it's immediately before the noun it describes, which isn't the case here.